foursight_core/decorators.py

```diff
@@ -5,13 +5,13 @@
 import os
 from dcicutils.misc_utils import ignored
 from functools import wraps
-from .check_schema import CheckSchema
-from .run_result import (
+from foursight_core.check_schema import CheckSchema
```

Comment:
Is this actually the preferred way to import rather than dot?

Reply:
Yes, the former will not work with unit testing as you are running from top level and thus searches for the files in that location (all relative imports have this weakness when invoking pytest from top level, so generally I avoid them)